study-builder/studies/atcp/contacting-physician.conf

```diff
@@ -0,0 +1,514 @@
+          "shownExpr": null,
+          "question": {
+            include required("../snippets/text-question.conf"),
+            "stableId": "PHYSICIAN_LASTNAME",
+            "questionType": "TEXT",
+            "inputType": "TEXT",
+            "isRestricted": false,
+            "isDeprecated": false,
+            "hideNumber": true,
+            "promptTemplate": {
+              "templateType": "HTML",
+              "templateText": "",
+              "variables": []
+            },
+            "additionalInfoHeaderTemplate": null,
+            "additionalInfoFooterTemplate": null,
+            "validations": [],
+            "suggestionType": "NONE",
+            "suggestions": [],
+            "placeholderTemplate": {
+              "templateType": "HTML",
+              "templateText": "$contacting_physician_prompt_lastname",
+              "variables": [
+                {
+                  "name": "contacting_physician_prompt_lastname",
+                  "translations": [
+                    {
+                      "language": "en",
+                      "text": ${i18n.en.contacting_physician_prompt_lastname}
+                    }
+                  ]
+                }
+              ]
+            },
+            "validations": [
+              {
+                "ruleType": "REQUIRED",
+                "hintTemplate": {
+                  "templateType": "TEXT",
+                  "templateText": "$contacting_physician_lastname_req_hint",
+                  "variables": [
+                    {
+                      "name": "contacting_physician_lastname_req_hint",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": ${i18n.en.contacting_physician_lastname_req_hint}
+                        }
+                      ]
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        },
+        {
+          "blockType": "QUESTION",
+          "shownExpr": null,
+          "question": {
+            include required("../snippets/text-question.conf"),
+            "stableId": "PHYSICIAN_MAIL",
+            "questionType": "TEXT",
+            "inputType": "TEXT",
+            "isRestricted": false,
+            "isDeprecated": false,
+            "hideNumber": true,
+            "promptTemplate": {
+              "templateType": "HTML",
+              "templateText": "",
+              "variables": []
+            },
+            "additionalInfoHeaderTemplate": null,
+            "additionalInfoFooterTemplate": null,
+            "validations": [],
+            "suggestionType": "NONE",
+            "suggestions": [],
+            "placeholderTemplate": {
+              "templateType": "HTML",
+              "templateText": "$contacting_physician_prompt_mail",
+              "variables": [
+                {
+                  "name": "contacting_physician_prompt_mail",
```

Comment:
Again, perhaps the address component could be used here. Now this would be a little bit trickier as our mail address component is tightly bound to user profiles in our system so it would require tweaking to be able to use here. At any rate, perhaps it makes sense here, perhaps not.